cppbuild/cppbuild.cmd

```diff
@@ -49,6 +49,13 @@ if not "%1"=="" (
         set "EXTRA_CMAKE_ARGS=!EXTRA_CMAKE_ARGS! -DGRADLE_WRAPPER=%2"
         echo "Setting -DGRADLE_WRAPPER=%2"
         shift
+    ) else if "%1"=="--parallel" (
```

Comment:
I've pushed a change on HEAD to run Windows build in parallel by default, i.e. it sets `CMAKE_BUILD_PARALLEL_LEVEL=%NUMBER_OF_PROCESSORS%` and supports a `--no-parallel` flag to run on a single CPU like the Linux build does.

Reply:
It's also useful to be able to specify a number of CPUs.  I've found running parallel, but leaving a couple of CPUs free lets the IDE and desktop remain responsive during builds.

Reply:
I've rebased with your changes and added `--parallel-cpus`.